panels/user-accounts/cc-user-panel.ui

```diff
@@ -655,4 +667,17 @@
       <widget name="last_login_row"/>
     </widgets>
   </object>
+  <object class="GtkPopover" id="parental_controls_unavailable_popover">
+    <property name="can_focus">False</property>
+    <property name="border_width">12</property>
+    <child>
+      <object class="GtkLabel">
+        <property name="visible">True</property>
+        <property name="can-focus">False</property>
+        <property name="wrap">True</property>
+        <property name="label" translatable="yes">Parental controls can now only be applied to non-administrator accounts. See the &lt;a href="https://support.endlessm.com/?TODO"&gt;FAQ entry&lt;/a&gt;.</property>
```

Comment:
This can cause bugs like https://phabricator.endlessm.com/T29270

Mind adapting the approach? : https://github.com/endlessm/gnome-control-center/blob/master/panels/info-overview/cc-info-overview-panel.ui#L169-L181

Reply:
Because translators could mis-translate the markup?

Reply:
Yes, I like to follow "no tags in the translatable string" policy; as long as we have got a alternative.

Reply:
I think that would make for a clunkier UI and would prefer to instead ensure that translators don’t break the markup (we already have to trust them to get format placeholders correct, and they can break an application just as easily if mis-translated).

Reply:
Ok, just a personal preference thing. :)